lib/src/utils/product.dart

```diff
@@ -1,17 +1,55 @@
 
 class Product {
-  String id;
-  String price;
-  String title;
-  String currencyCode;
+  String id = "";
+  String priceForAndroid ="";
+  String title ="";
+  String currencyCode = "";
+  String subscriptionPeriod = "";
+  double priceForIos =0;
```

Comment:
I'd rather use `late double` for such variables.

The issue is, that accessing `0` or `""` leads to undefined behavior (probably displaying 0 instead of a real price etc.). Accessing an uninitialized `late` throws an error, which can more easily be detected and fixed.

I'm valuing "guaranteed correctness if it doesn't throw" over "never throws", but that's personal preference.